### PR TITLE
Expand test coverage for mobile experience on fl

### PR DESCRIPTION
**Pull request by brangi**
## Description
issue:  https://github.com/department-of-veterans-affairs/va.gov-team/issues/10650

## Acceptance Criteria:

- [x] Tests cover the mobile experience
- [x] Tests pass with current functionality. 

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    });
+  });
+
+  it('should render in mobile layouts and tabs actions work', () => {
+    cy.visit('/find-locations');
+    cy.injectAxe();
+
+    // iPhone X
+    cy.viewport(400, 812);
+    cy.checks();
```
Comment (mpelzsherman) [inline src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js:45]: What does `checks` do?

**Comment by mpelzsherman** [comment]
What do you think about moving the `'should render the appropriate elements at each breakpoint'` test to the new mobile file?

File: src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import path from 'path';
+
+Cypress.Commands.add('checks', () => {
+  cy.axeCheck();
+
+  // Search
+  cy.get('#street-city-state-zip').type('Austin, TX');
+  cy.get('#facility-type-dropdown').select('VA health');
+  cy.get('#facility-search').click();
```
Comment (mpelzsherman) [inline src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js:9]: Maybe add a test that confirms the serviceType field behavior?

- disabled when non-community is selected
- enabled when community selected

File: src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
+import path from 'path';
+
+Cypress.Commands.add('checks', () => {
```
Comment (mpelzsherman) [inline src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js:3]: How about making this command more descriptive, e.g. `checkSearch`?

Comment (mpelzsherman) [reply src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js:3]: Also, does `Cypress.Commands.add` affect the global namespace for all Cypress tests, or just the file in which the command appears?

If it's the former, we should probably add a namespace prefix, e.g. `facilities.checkSearch`.

Comment (brangi) [reply src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js:3]: https://docs.cypress.io/api/cypress-api/custom-commands.html#Syntax

Comment (mpelzsherman) [reply src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js:3]: OK I was able to answer my own question. The answer appears to be that `Cypress.Commands.add()` only affects the current file, so there is no need to worry about namespacing.

To confirm this, I added a call to `cy.checks()` in the search test, then ran the mobile and search tests sequentially using the following command:

`❯ npx cypress run --config-file  config/cypress.json --spec src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js,src/applications/facility-locator/tests/e2e/search.cypress.spec.js`

I got this error:

`TypeError: cy.checks is not a function`

File: src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+      expect($element.width()).closeTo(48, 2);
+    });
+    cy.get('.desktop-map-container').should('exist');
+    cy.get('.react-tabs').should('not.exist');
+
+    // desktop - small
+    cy.viewport(1007, 1000);
+    cy.axeCheck();
+    cy.get('#facility-search').then($element => {
+      expect($element.width()).closeTo(899, 2);
```
Comment (mpelzsherman) [inline src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js:77]: FYI - I get a failure here when I run locally - on master as well.  Not sure why it seems to pass in CI.

**Review by mpelzsherman — APPROVED** [review]

**Pull request merged**